src/controllers/trainee/controller.ts

```diff
@@ -0,0 +1,92 @@
+class traineeController {
+    static instance: traineeController
+
+    static getInstance() {
+        if (traineeController.instance) {
+            return traineeController.instance
+        }
+        traineeController.instance = new traineeController();
+        return traineeController.instance;
+    }
+
+    get(req, res, next) {
+        try {
+            var console: Console
+            console.log("Inside get method of Trainee Controller");
+
+            res.send({
+                message: "Trainee fetched successfully",
+                data: [
+                    {
+                        name: "Trainee",
+                        address: "Noida"
+                    }
+                ]
+            });
+        }
+        catch (err) {
+            console.log("Inside err", err);
+        }
+    }
+
+    create(req, res, next) {
+        try {
+            var console: Console
+            console.log("Inside post method of Trainee Controller");
+
+            res.send({
+                message: "Trainee fetched successfully",
```

Comment:
update the message as per the operation

Reply:
Updated all the message as per the operation